### PR TITLE
strongswan pkg in zenbuild

### PR DESCRIPTION
**Pull request by srinibas-zededa**
added the strongswan package to zen build

### CODE REVIEW

File: pkg/strongswan/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            iptables \
+            iproute2 \
+			openssl \
+			openssl-dev \
+            curl\
+            curl-dev
+
+RUN  curl -Lo /tmp/strongswan/strongswan.tar.bz2 $STRONGSWAN_RELEASE 
+RUN  tar --strip-components=1 -C /tmp/strongswan -xjf /tmp/strongswan/strongswan.tar.bz2
+RUN  cd /tmp/strongswan
```
Comment (rvs) [inline pkg/strongswan/Dockerfile]: Why is this cd here if you have WORKDIR /tmp/strongswan above?

File: pkg/strongswan/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            --disable-md5 \
+            --disable-rc2 \
+            --disable-sha1 \
+            --disable-sha2 \
+            --disable-static
+RUN    make 
+RUN    make install DESTDIR=/out
+RUN    rm -rf /tmp/strongswan
+RUN    apk del build-base curl-dev openssl-dev
+RUN    rm -rf /var/cache/apk/*
```
Comment (rvs) [inline pkg/strongswan/Dockerfile]: Why are you running these rm and apk dell commands?

File: pkg/strongswan/Dockerfile (hunk 1 above)
Comment (srinibas-zededa) [reply pkg/strongswan/Dockerfile]: Sure, will change

File: pkg/strongswan/Dockerfile (hunk 2 above)
Comment (srinibas-zededa) [reply pkg/strongswan/Dockerfile]: Will remove

**Comment by rvs** [comment]
LGTM! Merged!

**Pull request merged**